packages/thumbnail-grid-block/src/ThumbnailGridBlock.spec.ct.tsx

```diff
@@ -54,14 +53,13 @@ describe('Thumbnail Grid Block', () => {
             .should('exist')
             .should('have.class', 'xs:tw-grid-cols-2 md:tw-grid-cols-3')
             .should('have.css', 'gap', '30px');
-        cy.get(ThumbnailItem).should('not.exist');
+        cy.get(ThumbnailItem).should('exist');
         cy.get(ThumbnailImage).should('not.exist');
         cy.get(ThumbnailImagePlaceholder).should('exist');
         cy.get(ThumbnailImageWrapper).should('have.class', 'tw-place-self-start');
-        cy.get(ThumbnailItemPlaceholder).should('have.class', 'tw-flex-row');
     });
 
-    it('should render one item if it is provided', () => {
+    it('should render one item if it is provided and automatically add an empty one', () => {
```

Comment:
Can you add a check that the second one is an empty one?

Reply:
Sure, done!